app/components/swatch.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { computed} from '@ember/object';
+import { htmlSafe } from '@ember/string';
+
+export default class Swatch extends Component{
+    @computed('shade','size')  
+    get style() {
+        const size = this.translateSize(this.size);
+        return htmlSafe(`background-color: ${this.shade}; width:${size}px; height:${size}px`);
```

Comment:
Related to my comment about an object for `sizes`, if you think you may want more control over the background color you could have an object that maps `shades` to specific hexcodes. I believe this approach would make it easier to  use colorscheme generators to create palettes with different selections of colors in the future.

Reply:
That is an awesome idea! The `Palette: Color Options` card is probably part of my next sprint 😏 but I'll def use that approach.